graphql_compiler/schema_generation/orientdb/schema_graph_builder.py

```diff
@@ -448,3 +468,67 @@ def _get_graphql_representation_of_non_graph_elements(non_graph_elements, inheri
             }
             graphql_reps[element_name] = GraphQLObjectType(element_name, fields, [])
     return graphql_reps
+
+
+def _get_indexes(index_data, elements):
+    """Return a set of IndexDefinitions describing the indexes defined in the OrientDB database."""
+    all_indexes = set()
+
+    # Get indexes from OrientDB.
+    all_class_names = set(elements.keys())
+
+    for index in index_data:
+        index_name = index['name']
+        index_type = index['type']
+        index_unique = index_type in UNIQUE_INDEX_TYPES
+        index_ordered = index_type in ORDERED_INDEX_TYPES
+        index_definition = index['indexDefinition']
+        index_base_classname = index_definition['className']
+        index_ignore_nulls = index_definition['nullValuesIgnored']
+
+        # Exclude indexes on non-graph classes (e.g. OUser).
```

Comment:
I'd call this orientdb metadata rather than non-graph classes, which mean something else in this repo